src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -154,143 +174,129 @@
+        sender-network-values            (rf/sub [:wallet/wallet-send-sender-network-values])
+        receiver-network-values          (rf/sub [:wallet/wallet-send-receiver-network-values])
+        tx-type                          (rf/sub [:wallet/wallet-send-tx-type])
+        unsupported-token-in-receiver?   (and (not= tx-type :tx/bridge)
+                                              (->> receiver-network-values
+                                                   (remove #(= (:type %) :add))
+                                                   (every? #(= (:type %) :not-available))))
+        suggested-routes                 (rf/sub [:wallet/wallet-send-suggested-routes])
+        routes                           (when suggested-routes
+                                           (or (:best suggested-routes) []))
+        no-routes-found?                 (and
+                                          (every-network-value-is-zero? sender-network-values)
+                                          (some? routes)
+                                          (not loading-routes?)
+                                          (not unsupported-token-in-receiver?))
+        receiver-networks                (rf/sub [:wallet/wallet-send-receiver-networks])
+        receiver-preferred-networks      (rf/sub [:wallet/wallet-send-receiver-preferred-networks])
+        receiver-preferred-networks-set  (set receiver-preferred-networks)
+        sending-to-unpreferred-networks? (some (comp not receiver-preferred-networks-set)
+                                               receiver-network-values)
+        input-error                      (controlled-input/input-error input-state)
+        limit-exceeded?                  (controlled-input/upper-limit-exceeded? input-state)
+        current-address                  (rf/sub [:wallet/current-viewing-account-address])
+        current-color                    (rf/sub [:wallet/current-viewing-account-color])
+        enough-assets?                   (rf/sub [:wallet/wallet-send-enough-assets?])
+        owned-eth-token                  (rf/sub [:wallet/token-by-symbol
+                                                  (string/upper-case constants/mainnet-short-name)
+                                                  enabled-from-chain-ids])
+        not-enough-asset?                (insufficient-asset-amount?
+                                          {:enough-assets?        enough-assets?
+                                           :token-symbol          token-symbol
+                                           :owned-eth-token       owned-eth-token
+                                           :input-state           input-state
+                                           :no-routes-found?      no-routes-found?
+                                           :limit-exceeded?       limit-exceeded?
+                                           :sender-network-values sender-network-values})
```

Comment:
Fixed this logic and moved it to a separate funciton, this namespace already has a lot of logic to handle. We must consider a refactor